tests/unit/test_zone.py

```diff
@@ -53,6 +53,61 @@ def test_rest_zone_retrieve(zones_config, zone, url):
     )
 
 
+@pytest.mark.parametrize("zone, url", [("test.zone", "zones/test.zone/versions")])
+def test_rest_zone_version_list(zones_config, zone, url):
+    z = ns1.rest.zones.Zones(zones_config)
+    z._make_request = mock.MagicMock()
+    z.list_versions(zone)
+    z._make_request.assert_called_once_with(
+        "GET",
+        url,
+        params={},
+        callback=None,
+        errback=None
+    )
```

Comment:
We don't check the results at all? Even to make sure that we get a 200 return code? :thinking:

Reply:
I followed the rest of the patterns for testing. I dont want to spend any more time on this SDK ~ the tests _and_ and the SDK are plainly foolish and I would wish to just replace them.